XRTK.Oculus/Packages/com.xrtk.oculus/Inspectors/BuildTools/OculusManifestPreprocessor.cs

```diff
@@ -19,76 +19,78 @@
 
 ************************************************************************************/
 
+using System;
 using UnityEngine;
 using UnityEditor;
 using System.IO;
 
-namespace XRTK.Oculus
+namespace XRTK.Oculus.Inspector.Build
 {
     public class OculusManifestPreprocessor
     {
         [MenuItem("Mixed Reality Toolkit/Tools/Create Oculus Quest compatible AndroidManifest.xml", false, 100000)]
         public static void GenerateManifestForSubmission()
         {
-            var so = ScriptableObject.CreateInstance(typeof(OculusUpdaterStub));
+            var so = ScriptableObject.CreateInstance(typeof(OculusEditorAssetLocator));
             var script = MonoScript.FromScriptableObject(so);
-            string assetPath = AssetDatabase.GetAssetPath(script);
-            string editorDir = Directory.GetParent(assetPath).FullName;
-            string srcFile = editorDir + "/AndroidManifest.OVRSubmission.xml";
+            var assetPath = AssetDatabase.GetAssetPath(script);
+            var editorDir = Directory.GetParent(assetPath).FullName;
+            var srcFile = $"{editorDir}/Build/AndroidManifest.OVRSubmission.xml";
```

Comment:
Great catch, I forgotten about this when I renamed the folder just now. 

```suggestion
            var srcFile = $"{editorDir}/BuildTools/AndroidManifest.OVRSubmission.xml";
```